buildpacks/ruby/CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
```

Comment:
The https://keepachangelog.com/en/1.0.0/ changelog style has category H3 headings like `### Added` etc - but there aren't any used in this changelog at the moment?

Reply:
I am not totally on board with the full keep-a-changelog (I love the project, but I don't use the full format in any of my projects). I prefer to have those annotations in line if they're required.

If we have a strong schema for our changelogs, we need to have validation on push to enforce them, or errors will build up over time.

I originally proposed using changelog annotations for buildpack release versioning, but AFAIK there are no plans for those annotations to be used in a meaningful way.

Reply:
One reason to at least adhere to the format strictly for now is the fact that the buildpacks I saw do. So that we have at least consistency. We can challenge the status quo of course, but I don't think this PR is the right battlefield. :)

Reply:
I updated it to use that format and also rev-d to use the latest version of the keep-a-changelog format: 1.1.0